lighthouse-core/aggregator/aggregate.js

```diff
@@ -77,6 +77,12 @@ class Aggregate {
       throw new Error(msg);
     }
 
+    // Audit resulted in an error, so doesn't contribute to score.
```

Comment:
Hmm, yea. 

I like the greyed out "-" icon for the failing ones. We should ditch the "coming soon" audits. They've been coming soon for.....ever. We could also use that icon for tests that we know are not passable for the user's setup (https://github.com/GoogleChrome/lighthouse/issues/1175#issuecomment-275476565) as a way to indicate something is up.